stories/profile/PriorityBillsCard.stories.tsx

```diff
@@ -10,4 +14,68 @@ export default createMeta({
   component: PriorityBillsCard
 })
 
-export const Primary = () => <PriorityBillsCard />
+const Template: ComponentStory<typeof PriorityBillsCard> = props => {
```

Comment:
I like the creativity here! I think to reflect the usage on the page, this should select the bill that the user clicks. See my comment about updating the onClick prop to make that easier